client/blocks/stats-navigation/intervals.scss

```diff
@@ -8,4 +8,9 @@
 			margin: 16px auto;
 		}
 	}
+	.gridicon {
+		display: none; /* Hidden by default */
+	}
+
```

Comment:
Following the previous comment, this wouldn't be needed if we only draw the icon on the selected row.